### DISCUSSION TITLE
Version of transformers model

### DISCUSSION THREAD

**Discussion by albin02t** [ThilinaRajapakse/simpletransformers#1205] [Q&A]
Hi Thilina, exceptional work on the library, made things a lot easier. Great to work with.
I just wanted to know which version of the transformers library does it use ?

**Comment by Zhylkaaa** [comment]
I guess you can check it out in setup.py under `install_requires`
```
install_requires=[
        "numpy",
        "requests",
        "tqdm>=4.47.0",
        "regex",
        "transformers>=4.2.0",
        "datasets",
        "scipy",
        "scikit-learn",
        "seqeval",
        "tensorboardx",
        "pandas",
        "tokenizers",
        "wandb>=0.10.32",
        "streamlit",
        "sentencepiece",
    ],
```
so current version requires 4.2.0 but I think this should be changed as it seems to me that BigBird model was added after 4.2.0
